tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compileOnSave": false,
   "compilerOptions": {
+    "importHelpers": true,
```

Comment:
What is this?

Reply:
![image](https://user-images.githubusercontent.com/43140629/52128472-0b6d9880-2603-11e9-97a1-517f3a694579.png)